src/factory-in-proc.js

```diff
@@ -131,6 +135,19 @@ class FactoryInProc {
     const node = new Node(options)
 
     series([
+      (cb) => node.exec.once('ready', cb),
```

Comment:
same comment as https://github.com/ipfs/js-ipfsd-ctl/pull/205#discussion_r173673672

Reply:
> Something isn't quite right here. the node is created but then what we listen on the ready event is exec??

@diasdavid Can you rephrase? `node.exec` is the IPFS instance being run by the daemon `node`. What's going on is that `new Node()` instantiates the daemon and returns immediately with a still-booting IPFS instance under `exec`. We could ask consumers to wait for their in-proc daemon's IPFS to be ready but that is inconsistent with the go and js factories, which deliver ready instances.